script/Ciel/Day 14/14_01B_01_CIEL14B_1.txt

```diff
@@ -121,27 +129,27 @@ If I were to truly think about Arcueid―――
 [sha:fd3e4c80e7012fef51b7f5a8aff01083eb6b35a9]{
 -- Page 10, Offset 38337.
 -- ……額が熱い。
-...My forehead is hot.
+My forehead is burning up...
 }
 [sha:6945d24113dfe31f2fba39e40a1e65c248b990c3]{
 -- Page 10, Offset 38338.
 -- 脳がぐつぐつと高温に熔ける中、確かに―――
-But while my brain was simmering away from the heat―――
+But even though my brain is beginning to boil from the heat―――
 }
 [sha:95d00f7905e5cf412361950f80ea8b138ec57db2]{
 -- Page 10, Offset 38339.
 -- たった一本だけの、死に易い『線』を見た。
-I'm sure I saw one, and only one, 'line' exposed with death.
+I'm sure I saw one――a 'line' where death comes easily.
 }
 [sha:67057dd51a14c527772a322a993aefaeb32309f8]{
 -- Page 11, Offset 38340.
 -- 「けど、やっぱりお断りだ。
-"But I'll have to refuse.
+"But I'm afraid I have to refuse after all.
 }
 [sha:dc05b49a0b8b4f1d4697921085c2de76bd485a4d]{
 -- Page 11, Offset 38341.
 -- 俺が惚れてるのはシエルであって、アルクェイドじゃないんだから……！」
-Because the one I love is Ciel, and not you...!"
+Because the one I love is Ciel, not you!"
```

Comment:
the length of the line, and his obvious vocalization and emphasis on "Arcueid" makes me feel like it has to be included.
>not you, Arcueid!